### DISCUSSION TITLE
Erro ao assinar nfse

### DISCUSSION THREAD

**Discussion by sidneiJr1984** [TadaSoftware/PyNFe#323] [Q&A]
boa noite estou comecando a usar a lib mas estou recebendo um erro ao tentar consultar uma rps sera que voces poderiao me auxiliar

o arquivo NFSE.xml eh gerado
![erro](https://github.com/TadaSoftware/PyNFe/assets/45664938/7cec8751-cabc-49f2-8573-44ca9892682e)

**Comment by Natutechbi** [comment]
Eu tbm estou com este problema

**Comment by juniortada** [comment]
Foi corrigido neste PR
https://github.com/TadaSoftware/PyNFe/pull/405
